src/smc-webapp/r_misc/selector-input.tsx

```diff
@@ -7,7 +7,8 @@ interface Props {
   options:
     | string[]
     | { value: string; display: JSX.Element }[]
-    | { [keys: string]: JSX.Element };
+    | { [keys: string]: JSX.Element }
+    | Readonly<{ [keys: string]: string }>;
```

Comment:
this is because of `project/ask-filename.tsx`. to check, use the new filename dialog and click on the selector how to generate new names